signals.c

```diff
@@ -0,0 +1,72 @@
+/*
+  Not part of Grbl. KeyMe specific.
+  
+  Signal is one layer of abstraction above adc.h and adc.c.
+
+  ADC values are read in specific time intervals, filtered and
+  stored in the appropriate arrays.
+*/
+
+#include "signals.h"
+#include "adc.h"
+#include "systick.h"
+
+#define N_FILTER 3
+#define SIGNALS_CALLBACK_INTERVAL 20
```

Comment:
Yes, this value is in milliseconds, so we would be sampling at 50 Hz, which might be too slow. The sampling frequency depends on how fast we move the gripper while sampling because that will affect how fast the ADC value will change. I am open to changing it.